test/base/LoanTermsConsensusProcessRequestTest.js

```diff
@@ -167,5 +170,79 @@ contract('LoanTermsConsensusProcessRequestTest', function (accounts) {
                 assert.equal(error.reason, expectedErrorMessage);
             }
         })
-    })
+    });
+
+    withData({
+        _9_nonce_taken: [
+            1, 320, [responseOne], true, 'REQUEST_NONCE_TAKEN'
```

Comment:
I think this scenario is already covered on the branch develop here https://github.com/teller-protocol/teller-protocol-v1/blob/develop/test/base/LoanTermsConsensusProcessRequestTest.js#L108
Please review it is covering the same scenario.

Reply:
Fixed